agent_payload.proto

```diff
@@ -59,6 +61,28 @@ message EventsPayload {
 	CommonMetadata metadata = 2;
 }
 
+message SketchPayload {
+        message Summary {
+                message Sketch {
+                        message Entry {
+                                double v = 1;
+                                int64 g = 2;
+                                int64 delta = 3;
+                        }
```

Comment:
this might also be a good time to remove min and max by including them in the `Entries` array